### ISSUE TITLE
Keep broadcasting after loosing focus

### ISSUE THREAD

**Issue by UndeadKernel** [Eugeny/terminus#8800]
**Is your feature request related to a problem? Please describe.**
With the current broadcast mechanism, if the terminal looses focus, broadcasting is stopped when regaining focus.

**Describe the solution you'd like**
It'd be fantastic if broadcasting is kept when focus is lost.

**Describe alternatives you've considered**
Maybe there could be a setting option to have a "sticky" broadcasting functionality.

**Comment by lavinnyv** [comment]
> **Is your feature request related to a problem? Please describe.** With the current broadcast mechanism, if the terminal looses focus, broadcasting is stopped when regaining focus.
> 
> **Describe the solution you'd like** It'd be fantastic if broadcasting is kept when focus is lost.
> 
> **Describe alternatives you've considered** Maybe there could be a setting option to have a "sticky" broadcasting functionality.

i would like to request this as well

**Comment by DennisSkov** [comment]
I agree. I switched from MobaXTerm, and the broadcast feature was the only reason I switched to Tabby. MobaXTerm had it as well but it worked in a different way - you would not automatically stop being in "Multi-Exec" mode. 

A setting to control the behavior for broadcasting to the selected pane would be great - and also a setting for the behavior when broadcasting to _all_ tabs. 

MobaXTerm also had the option to exclude certain sessions from broadcasting - that would be neat as well, although I can live without.

**Comment by nproctor-dev** [comment]
This is exactly the workflow that pushed me to build tabby-multiexec.

I wanted MobaXterm-style MultiExec where broadcast remains enabled until I explicitly turn it off, rather than losing the mode when I click into another pane.

The plugin also adds dynamic per-pane Exclude controls, so a pane can be removed from the broadcast set without stopping/restarting MultiExec. You can change the active/source pane while MultiExec remains enabled as well.

Docs/screenshots:
https://github.com/nproctor-dev/tabby-multiexec-support

Disclosure: I’m the plugin author. I found this issue after building it, but the behavior described here — persistent broadcast plus session exclusion — is specifically what I was trying to solve.

<blockquote><img src="https://opengraph.githubassets.com/1f082dbb9590b9dd3128ca0514c7846eff07114766de969f71b08b85e36daa83/nproctor-dev/tabby-multiexec-support" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/nproctor-dev/tabby-multiexec-support">GitHub - nproctor-dev/tabby-multiexec-support: Documentation, support, and issue tracking for Tabby MultiExec</a></strong></div><div>Documentation, support, and issue tracking for Tabby MultiExec - nproctor-dev/tabby-multiexec-support</div></blockquote>
